modules/virt-customizing-storage-profile.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc
+
+[id="virt-customizing-storage-profile_{context}"]
+
+= Customizing the storage profile
+
+Customize your storage profile to define default settings for data volumes that you frequently allocate. This can be helpful if, for example, a storage class is recognized by the CDI as unsupported. In this case, the administrator can set appropriate values in the storage profile to ensure successful allocation.
```

Comment:
is not recognized by CDI

Reply:
How's this:

This can be helpful if, for example, a storage class is not recognized by the CDI, or recognized as unsupported.